lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -1742,6 +1742,11 @@ def setUp(self):
         for student in self.students:
             CourseEnrollment.enroll(student, self.course.id)
 
+        self.students_who_may_enroll = self.students + [UserFactory() for _ in xrange(6, 11)]
+        for student in self.students_who_may_enroll:
+            CourseEnrollmentAllowed.objects.create(
+                email=student.email, course_id=self.course.id)
```

Comment:
Closing paren should always be on the next line (you have a lot of this throughout the PR you'll need to fix up). Please see https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines